.scripts/unit.js

```diff
@@ -49,7 +49,6 @@ const webpackDevServerHandler = (data) => {
       cleanupDevServer();
       process.exit(0);
     }).catch((err) => {
-      console.error(err);
```

Comment:
This error message winds up being useless--basically telling us what we already know from the subprocess output